src/platforms/android/migration.mdx

```diff
@@ -10,7 +10,45 @@ description: "Migrating between versions of Sentry's SDK for Android."
 
 The `io.sentry.android.gradle` >= `3.0.0` requires [Android Gradle Plugin >= 7.0.0](https://developer.android.com/studio/releases/gradle-plugin#7-0-0).
 
-The `tracingInstrumentation` feature requires the Sentry's Android SDK version `4.0.0` and above, otherwise it throws a `NoClassDefFoundError` exception.
```

Comment:
removed this, because SAGP is now capable of checking the main SDK version and disabling that functionality to prevent runtime crashes